data/expected_cil/parent_call.cil

```diff
@@ -0,0 +1,151 @@
+(class x25_socket (ioctl read write create getattr setattr lock relabelfrom relabelto append map bind connect listen accept getopt setopt shutdown recvfrom sendto name_bind))
+(class xdp_socket (ioctl read write create getattr setattr lock relabelfrom relabelto append map bind connect listen accept getopt setopt shutdown recvfrom sendto name_bind))
+(classorder (alg_socket anon_inode appletalk_socket association atmpvc_socket atmsvc_socket ax25_socket binder blk_file bluetooth_socket bpf caif_socket can_socket cap2_userns cap_userns capability capability2 chr_file dbus dccp_socket decnet_socket dir fd fifo_file file filesystem icmp_socket ieee802154_socket infiniband_endpoint infiniband_pkey ipc ipx_socket irda_socket isdn_socket iucv_socket kcm_socket kernel_service key key_socket llc_socket lnk_file lockdown memprotect msg msgq netif netlink_audit_socket netlink_connector_socket netlink_crypto_socket netlink_dnrt_socket netlink_fib_lookup_socket netlink_generic_socket netlink_iscsi_socket netlink_kobject_uevent_socket netlink_netfilter_socket netlink_nflog_socket netlink_rdma_socket netlink_route_socket netlink_scsitransport_socket netlink_selinux_socket netlink_socket netlink_tcpdiag_socket netlink_xfrm_socket netrom_socket nfc_socket node node_socket packet packet_socket peer perf_event phonet_socket pppox_socket process process2 qipcrtr_socket rawip_socket rds_socket rose_socket rxrpc_socket sctp_socket security sem service shm smc_socket sock_file socket system tcp_socket tipc_socket tun_socket udp_socket unix_dgram_socket unix_stream_socket vsock_socket x25_socket xdp_socket))
+(sensitivity s0)
+(sensitivityorder (s0))
+(user system_u)
+(role system_r)
+(role object_r)
+(userrole system_u system_r)
+(userrole system_u object_r)
+(userlevel system_u (s0))
+(userrange system_u ((s0) (s0)))
+(handleunknown allow)
+(typeattribute domain)
+(typeattribute resource)
+(typeattribute abc)
+(typeattributeset resource (abc))
+(typeattribute bar)
+(typeattributeset resource (bar))
+(type dom)
+(roletype system_r dom)
+(typeattributeset domain (dom))
+(type kernel_sid)
+(roletype system_r kernel_sid)
+(typeattributeset domain (kernel_sid))
+(type qwe)
+(roletype system_r qwe)
+(typeattributeset domain (qwe))
+(type security_sid)
+(roletype object_r security_sid)
+(typeattributeset resource (security_sid))
+(type unlabeled_sid)
+(roletype object_r unlabeled_sid)
+(typeattributeset resource (unlabeled_sid))
+(type foo)
+(roletype object_r foo)
+(typeattributeset bar (foo))
+(typeattributeset resource (foo))
+(type xyz)
+(roletype object_r xyz)
+(typeattributeset abc (xyz))
+(typeattributeset resource (xyz))
+(macro abc-read ((type this) (type source)) (allow source this (file (read))))
+(macro bar-read ((type this) (type source)) (allow source this (file (read))))
+(macro foo-read ((type this) (type source)) (allow source this (dir (read))))
+(macro xyz-read ((type this) (type source)) (call abc-read (abc source)))
+(call foo-read (bar dom))
```

Comment:
Discussed on call.  I think this should probably be "(call bar-read (foo dom))" instead.